UPGRADE-1.12.md

```diff
@@ -84,24 +84,20 @@ with test or remove these services with complier pass.
 9. Passing a `Gaufrette\Filesystem` to `Sylius\Component\Core\Uploader\ImageUploader` constructor is deprecated since
 Sylius 1.12 and will be prohibited in 2.0. Use `Sylius\Component\Core\Filesystem\Adapter\FlysystemFilesystemAdapter` instead.
 
-### Asset management changes
+### Frontend toolset changes
+
+#### Dependencies update
+
+In `1.12` we have updated our frontend dependencies to the latest versions. This means that you might need to update your dependencies as well.
+The full list of all dependencies can be found in the [package.json](./package.json) file.
+
+Due to a fact every project is different, we cannot provide a list of all changes that might be needed. However, we have prepared a list of the most common changes that might be needed.
```

Comment:
```suggestion
Because every project is different, we cannot provide a list of all changes that might be needed. However, we have prepared a short list of fixes for the most common issues.
```